lod/whats-new.md

```diff
@@ -8,6 +8,38 @@
 
 # What's New in Lab On Demand
 
+## **Released June 12, 2020**
+
+**IDLx Replacements**
+- There is a new option in the IDLx @lab menu for Replacements. Replacements allow for text or regex find and replace. Replacements can be used to cleanup or extend existing instructions sources, potentially eliminating  the need for lab errata. 
+
+**Azure CSS Improvements**
+- Azure CSS Resource Group names have been simplified to be read more user friendly.
+- Removed the Name field from Azure Enrollment Accounts. Only the Principal Name is displayed.
+ 
+**LOD Improvements and Fixes**
+
+- Per hour pricing is now displayed on lab profile details pages and returned in a LabProfile and Catalog API calls. 
+- When a cloud based lab inherits the environment from another lab, the virtualization platform and cloud platform are now disabled.
```

Comment:
Would it be better if this was "When a cloud based lab inherits from another lab, the virtualization and cloud platforms are now disabled"

Reply:
Yeah, I’ll change that.